api/models/build-task.js

```diff
@@ -33,6 +34,9 @@ const associate = ({
     foreignKey: 'buildTaskTypeId',
     allowNull: false,
   });
+  BuildTask.belongsTo(SiteBuildTask, {
+    foreignKey: 'siteBuildTaskId',
```

Comment:
Since we don't explicitly set `allowNull: false`, does that mean we allow null on build task's site build task id?

Reply:
Yeah I was allowing it for now unless we want to go back and clear out old build tasks (assigning them retroactively is possible but tough)

Reply:
That's what I thought but just wanted to verify. I think it's fine to have old build tasks with no site id in the record.